### PR TITLE
headers-userhome-ui

### PR DESCRIPTION
**Pull request by tammytamayao**
- created mainheader file under headers folder
- added ui for userhome

### CODE REVIEW

File: front-end/src/index.css [hunk 1]
```diff
@@ -95,4 +95,9 @@ code {
   .form-input {
     @apply w-full bg-white rounded border border-gray-300 focus:border-indigo-500 focus:ring-2 focus:ring-indigo-200 text-base outline-none text-gray-700 py-1 px-3 leading-8 transition-colors duration-200 ease-in-out
   }
+
+  /* headers */
+  .btn-header {
+    @apply mr-5 text-white hover:text-gray-900
+  }
 }
```
Comment (rafael-parayno) [inline front-end/src/index.css]: add new line

**Review by rafael-parayno — CHANGES_REQUESTED** [review]

File: front-end/src/index.css (hunk 1 above)
Comment (tammytamayao) [reply front-end/src/index.css]: Done

File: front-end/src/components/UserHome.tsx [hunk 2]
```diff
@@ -28,21 +22,25 @@ const UserHome = () => {
       console.log(res);
```
Comment (rafael-parayno) [inline front-end/src/components/UserHome.tsx]: remove this console. All console.log should be remove for prod ready.

**Review by rafael-parayno — CHANGES_REQUESTED** [review]

File: front-end/src/components/UserHome.tsx (hunk 2 above)
Comment (tammytamayao) [reply front-end/src/components/UserHome.tsx]: Done

**Review by rafael-parayno — APPROVED** [review]

**Pull request merged**